packages/datadog-instrumentations/src/pg.js

```diff
@@ -30,9 +30,9 @@ function wrapQuery (query) {
     const callbackResource = new AsyncResource('bound-anonymous-fn')
     const asyncResource = new AsyncResource('bound-anonymous-fn')
     const processId = this.processID
-    let pgQuery = {
-      text: arguments[0]
-    }
+    let pgQuery = typeof arguments[0] === 'object' ? {
```

Comment:
Why clone the object instead of passing it as-is? It seems like before the regression it wasn't cloned.

Reply:
As mentioned, I don't totally understand why this was a mutable let in the first place or why we are overwriting this on line 57 from the queue (or what the queue is). It feels like an overloaded variable with two totally separate uses. So was trying to as closely preserve the original authors intent here. Feel free to update this though or take the findings and do something different with them too (there's an alternative approach I describe as well)! I'm not wedded to the specifics, was kind of just trying to show the specific issue.

Reply:
Looks like a fix for this has actually landed in #3087